extensions/brb.md

```diff
@@ -0,0 +1,93 @@
+---
+title: IRCv3 `brb` Extension
+layout: spec
+extends: resume
+copyrights:
+  -
+    name: "Jess"
+    period: "2019"
+    email: "jess@jesopo.uk"
+---
+
+## Notes for implementing work-in-progress version
+
+This is a work-in-progress specification.
+
+Software implementing this work-in-progress specification MUST NOT use the
+unprefixed `brb` capability name. Instead, implementations SHOULD
+use the `draft/brb` capability name to be interoperable with other
+software implementing a compatible work-in-progress version.
+
+The final version of the specification will use an unprefixed capability name.
+
+## Introduction
+
+Occasionally, clients will need to restart in order to update software and, at
+the moment, this will cause a `QUIT` to be sent out for the user on all their 
+networks (unless the client supports upgrading in-place)
+
+With the introduction of the `resume` spec, we have an opportunity to leverage 
+the ability to resume an old session to achieve this but we need a way to 
+client-initiate a clean disconnect that holds a session open (`resume` is 
+designed solely for ping timeouts.)
+
+## Dependencies
+
+Clients wishing to use this capability MUST also support the `draft/resume-0.3`
+capability and the `FAIL` message.
+
+## Architecture
+
+### Commands
+
+#### `BRB` Command
+
+This command MAY ONLY be sent by a client that has previously negotiated the 
+`resume` capability.
```

Comment:
"has previously negotiated the `resume` capability, and has received and stored their resume token"

just to make it clear that they need to get and keep the token in order for the BRB to be effective